apps/music/js/metadata/id3v2.js

```diff
@@ -140,49 +140,63 @@ var ID3v2Metadata = (function() {
    * @param {Metadata} metadata The (partially filled-in) metadata object.
    */
   function parseFrame(header, blobview, metadata) {
-    var frameid, framesize, frameflags, frame_unsynchronized = false;
+    var frameid, framesize, frameflags, frame_unsynchronized = false,
+        data_len_indicator = false, unsupported_flags = false;
 
     switch (header.version) {
     case 2:
       frameid = blobview.readASCIIText(3);
       framesize = blobview.readUint24();
       frameflags = 0;
+      unsupported_flags = (frameflags !== 0);
```

Comment:
given that you just set frameflags to 0, can you just do unsupported_flags = false here?